pages/common/ssh-copy-id.md

```diff
@@ -0,0 +1,11 @@
+# ssh-copy-id
+
+> Install your public key in a remote machine's authorized_keys.
+
+- Copy your key to a remot machine:
```

Comment:
What's the default if there are several local keys available? Also, typo: "remot" --> "remote".